### PR TITLE
Adding exceptions

### PR DESCRIPTION
**Pull request by Rumyra**

### CODE REVIEW

File: files/en-us/web/api/csskeywordvalue/csskeywordvalue/index.html [hunk 1]
```diff
@@ -25,6 +25,13 @@ <h3 id="Parameters">Parameters</h3>
  <dd>Sets or returns the value of the new <code>CSSKeywordValue</code>.</dd>
 </dl>
 
+<h3 id="Exceptions">Exceptions</h3>
+
+<dl>
+	<dt><code>TypeError</code></dt>
+  <dd>If the <code>value</code> parameter is not specified or it is not of type {{jsxref('String')}}.</dd>
```
Comment (sideshowbarker) [inline files/en-us/web/api/csskeywordvalue/csskeywordvalue/index.html:32]:
```suggestion
  <dt><code>TypeError</code></dt>
  <dd>If the <code>value</code> parameter is not specified or it is not of type {{jsxref('String')}}.</dd>
```

**Review by sideshowbarker — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by sideshowbarker** [comment]
@Aayushap1 Could you please stop approving PRs? Especially could you stop doing without adding any comment, and approving the same PR multiple times, as you did for this PR? And also especially could you stop doing it for PRs that somebody else has already approved?

Maybe you don’t realize it, but every time you approve a PR as you did here, GitHub sends out a notification to multiple people. So the effect of what you’ve been doing is just to create a lot of extra notifications that are essentially just noise. We’re all already getting a lot of notifications, so that kind of noise is really not helpful.

**Review by Elchi3 — APPROVED** [review]
Sorry for not seeing this earlier!

**Comment by Rumyra** [comment]
To be honest @Elchi3 I hadn't thought about checking on my own prs until now 😆 - thank you!

**Pull request merged**